src/fits_schema/binary_table.py

```diff
@@ -399,6 +399,7 @@ class String(Column):
     def __init__(
         self,
         *,
+        string_size: int = None,
```

Comment:
length and size is inconsistent, right? Or do you attach some semantics to that?